application/actions/requests_test.go

```diff
@@ -1273,3 +1273,197 @@ func (as *ActionSuite) verifyRequestCreateInput(ctx context.Context, input api.R
 
 	as.Equal(string(input.Visibility), string(request.Visibility), "Visibility is not correct")
 }
+
+func (as *ActionSuite) Test_requestsUpdate() {
+	f := createFixturesForRequestQuery(as)
+
+	goodRequest := api.RequestUpdateInput{
+		Description: nulls.NewString("new description"),
+	}
+
+	badRequestField := api.RequestCreateInput{
+		OrganizationID: f.Organization.UUID,
+	}
+
+	empty := ""
+	badRequestData := api.RequestUpdateInput{
+		Title: &empty,
+	}
+
+	tests := []struct {
+		name       string
+		user       models.User
+		input      interface{}
+		request    models.Request
+		wantStatus int
+	}{
+		{
+			name:       "authn error",
```

Comment:
```suggestion
			name:       "authz error",
```

Reply:
Are you sure? Even though the http status is "Unauthorized" (which I am thinking is not totally accurate) I think this is an authn error. Not that it matters much, but I'm scratching my head over this one.

Reply:
I was just looking at the wantStatus. It's not worth any more time on it. So, feel free to leave it as is.